### ISSUE TITLE
How can I get an image resolution greater than 256?

### ISSUE THREAD

**Issue by gofixyourself** [PeterouZh/CIPS-3D#14]
Hi! You did a great job, thanks for such a great paper and promptly published CIPS-3D code.

I've already gotten good results with your pipeline, but for images with resolution 64x64. Now I'm waiting the results of generating images with a resolution of 128x128. And I will further train for higher resolution images.

I understand correctly, in order to get 512x512 images, I need to convert the original FFHQ dataset once again through your script dataset_tool.py, but specifying the resize for 512 in it? And after I need to run training pipeline with lower values for generator learning rate and discriminator learning rate? 

Thanks!

**Comment by PeterouZh** [comment]
Yes, the pipeline is right. Some hyperparameters may also need to be adjusted.

**Comment by shoutOutYangJie** [comment]
@gofixyourself  DO you get the results of 512x512?  how does it performance?

**Comment by shoutOutYangJie** [comment]
> Yes, the pipeline is right. Some hyperparameters may also need to be adjusted.

Can you share the command line about how to train 256x256?

**Comment by PeterouZh** [comment]
> > Yes, the pipeline is right. Some hyperparameters may also need to be adjusted.
> 
> Can you share the command line about how to train 256x256?

Please refer to `exp/cips3d/bash/ffhq_exp`.
